paymentsheet/src/test/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEventTest.kt

```diff
@@ -163,6 +166,7 @@ class PaymentSheetEventTest {
 
         val expectedConfig = mapOf(
             "customer" to false,
+            "customer_access_provider" to null,
```

Comment:
Should we omit the key if the value is null?

Reply:
iOS is choosing to provide a [null value](https://github.com/stripe/stripe-ios/blob/c84adf2135b29b4b63c39f50af8a8ef682b9fea8/StripePaymentSheet/StripePaymentSheet/Source/Analytics/PaymentSheetAnalyticsHelper.swift#L326) as well. I'm going to verify with John to make sure they don't auto filter out null values though in their analytics.